pkg/relayer/proxy/proxy.go

```diff
@@ -44,7 +45,7 @@ type relayerProxy struct {
 	// providedServices is a map of the services provided by the relayer proxy. Each provided service
 	// has the necessary information to start the server that listens for incoming relay requests and
 	// the client that proxies the request to the supported native service.
-	providedServices map[string][]*ProvidedService
+	providedServices map[string][]RelayServer
```

Comment:
Should  this name be updated as well?

```suggestion
	relayServers map[string][]RelayServer
```